arm-network/2017-03-01/swagger/vmssPublicIpAddress.json

```diff
@@ -0,0 +1,252 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2017-03-01"

```

Comment:
can you change this to 2017-03-30 and see what happens please? this was suggested by sdk team as workaround but not sure if it is still a problem. Resources.azure.com have a problem with this